tests/test-cases/Group1-Docker-Commands/1-4-Docker-Create.robot

```diff
@@ -33,6 +33,21 @@ Create with volume
     #Should Be Equal As Integers  ${rc}  0
     #Should Not Contain  ${output}  Error
 
+Create with named volume
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} create -v test-named-vol:/testdir busybox
+    Should Be Equal As Integers  ${rc}  0
+    Should Not Contain  ${output}  Error
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} start ${output}
+    Should Be Equal As Integers  ${rc}  0
+    Should Not Contain  ${output}  Error
+
+    ${status}=  Get State Of Github Issue  1718
```

Comment:
Sure, sounds good. I'll create a new issue for filtering volumes by name because a fix for #1718 may not include name filtering since Docker API v1.23 doesn't support it.